src/Oxpecker/ModelValidation.fs

```diff
@@ -0,0 +1,137 @@
+namespace Oxpecker
+
+[<AutoOpen>]
+module ModelValidation =
+
+    open System.Collections.Generic
+    open System.ComponentModel.DataAnnotations
+    open System.Runtime.CompilerServices
+    open Microsoft.AspNetCore.Http
+
+    type ValidationErrors(errors: ResizeArray<ValidationResult>) =
+        let errorDict =
+            lazy
+                (let dict = Dictionary<string, ResizeArray<string|null>>()
```

Comment:
Do we really need to do this nullable?

Reply:
Yes
![image](https://github.com/user-attachments/assets/21788d29-58cf-42e1-9769-c217b9b6fbad)